apps/server/src/modules/user-import/controller/import-user.controller.ts

```diff
@@ -135,4 +137,19 @@ export class ImportUserController {
 	async populateImportUsers(@CurrentUser() currentUser: ICurrentUser): Promise<void> {
 		await this.userImportFetchUc.populateImportUsers(currentUser.userId);
 	}
+
+	@Post('cancel')
+	@ApiOperation({
+		summary: 'Cancel migration wizard',
+		description: 'Cancel current migration process',
+	})
+	@ApiNoContentResponse()
+	@ApiCreatedResponse()
+	@ApiUnauthorizedResponse()
+	@ApiServiceUnavailableResponse()
```

Comment:
I don't think there is a ServiceUnavailable here. Also you have NoContent instead of Created.